books.js

```diff
@@ -0,0 +1,82 @@
+const bookSection = document.querySelector('.books');
+const bookTitle = document.getElementById('book-title');
+const bookAuthor = document.getElementById('book-author');
+const form = document.getElementById('form');
+
+function getBooks() {
+  let myBooks;
+  if (localStorage.getItem('newBooks') === null) {
+    myBooks = [];
+  } else {
+    myBooks = JSON.parse(localStorage.getItem('newBooks'));
+  }
+  return myBooks;
+}
+
+// Div that contains book HTML
+function showBooks(book) {
+  const bookList = `
+  <div id="book-info">
+  <p>${book.title}</p>
+  <p>${book.author}</p>
+  <button type="submit" class="remove-book">Remove</button>
+  <hr>
+  </div>
+  `;
+
+  bookSection.innerHTML += bookList;
+}
+
+// This code clears the input field after books have been submitted
+function clearInput() {
+  bookTitle.value = '';
+  bookAuthor.value = '';
+}
+
+// Add books to local storage
+function setLocalStorage(item) {
+  const books = getBooks();
+  books.push(item);
+
+  localStorage.setItem('newBooks', JSON.stringify(books));
+}
+
+// Add New Book To List
+
+function addBookToList(e) {
+  e.preventDefault();
+  if (bookTitle.value === '' && bookAuthor.value === '') {
+    return;
+  }
+  const newBook = {
+    title: bookTitle.value,
+    author: bookAuthor.value,
+  };
+  showBooks(newBook);
+  clearInput();
+  setLocalStorage(newBook);
+}
+
+form.addEventListener('submit', addBookToList);
+
+// Remove books from Local Storage
+
+function removeFromLocalStorage() {
+  const myBooks = JSON.parse(localStorage.getItem('newBooks'));
+  myBooks.forEach((index) => {
+    myBooks.splice(index, 1);
+    return myBooks;
+  });
+  localStorage.setItem('newBooks', JSON.stringify(myBooks));
+}
+
+// Remove Books
+function deleteBook(book) {
+  if (book.classList.contains('remove-book')) {
+    book.parentElement.remove();
+  }
+}
+bookSection.addEventListener('click', (e) => {
+  deleteBook(e.target);
+  removeFromLocalStorage(e.target);
```

Comment:
Calling The function `removeFromLocalStorage(e.target)` with `e.target` as an argument, while in the function definition in line `64` there is no way to handle that argument.

Ex: 
```javascript
function sum(num1, num2) {
 const result = parameter1 + parameter2
 return result
}

sum(1,2)
```
1 and 2 is the function `sum()` arguments, while num1 , num2 are the function parameters. 

Right now the App works fine to Delete the books from the page (DOM), however, if you check the `localStorage`  in the console, after removing a number of books.. you will notice that the localStorage is not aligned with the same result as in your App.

Please fix `removeFromLocalStorage` function so when you remove a book from the app, it gets removed from the page and also from your books collection.

Reply:
Hi @od-c0d3r, 
Thank you for the review.
I have been able to fix things and I'm sending it in for review now.

Reply:
This issue is well resolved! ✔️